src/SpaceWatcher.ts

```diff
@@ -10,18 +10,19 @@ import { APP_PLAYLIST_REFRESH_INTERVAL } from './constants/app.constant'
 import { TWITTER_AUTHORIZATION } from './constants/twitter.constant'
 import { Downloader } from './Downloader'
 import { logger as baseLogger } from './logger'
-import { SpaceChat } from './SpaceChat'
+import { Twitter } from './namespaces/Twitter'
+import { SpaceCaptions } from './SpaceCaptions'
 import { Util } from './Util'
 
 export class SpaceWatcher extends EventEmitter {
   private logger: winston.Logger
-  private metadata: Record<string, any>
-  private liveStreamStatus: Record<string, any>
+  private metadata: Twitter.AudioSpaceMetadata
```

Comment:
If you are exporting it from a file named `Twitter` you should not have extra `Twitter` object here.